library/VTKExtensions/Rendering/vtkF3DRenderer.h

```diff
@@ -175,6 +177,9 @@ class vtkF3DRenderer : public vtkOpenGLRenderer
   std::string HDRIFile;
   std::string FontFile;
 
+  vtkNew<vtkLightKit> LightKit;
```

Comment:
Isn't there already a default light kit in all scenes ? What about Renderer->GetLights() ?

Reply:
According to the vtkLightKit doc

> To enforce the relationship between the three lights, the intensity of the fill and headlights are set as a ratio to the key light brightness. Thus, the brightness of all the lights in the scene can be changed by changing the key light intensity.

so having a handle on a full light kit rather than individual lights would make changing the intensity consistently easier I would guess? (disclaimer: again, I know nothing about f3d or VTK)

Reply:
tbh @snoyer , for a F3D and VTK beginner, this is great code !

We will check that and get back to you.